.github/workflows/build-win.yml

```diff
@@ -0,0 +1,151 @@
+name: Build Windows
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+    - main
+    - release/*
+  pull_request:
+    branches:
+    - main
+    - release/*
+
+concurrency:
+  # Cancel any workflow currently in progress for the same PR.
+  # Allow running concurrently with any other commits.
+  group: build-${{ github.event.pull_request.number || github.sha }}
```

Comment:
Need to change this and the other one to be unique, otherwise they will cancel each other.

Reply:
Yeah, I am aware of that. Fixing a bunch of other stuff now.